angular.json

```diff
@@ -183,14 +269,14 @@
             "devServerTarget": "angular-ssr-example:serve"
           },
           "configurations": {
-            "production": {
-              "devServerTarget": "angular-ssr-example:serve:production"
+            "prod": {
+              "devServerTarget": "angular-ssr-example:serve:prod"
             },
-            "certification": {
-              "devServerTarget": "angular-ssr-example:serve:certification"
+            "cert": {
+              "devServerTarget": "angular-ssr-example:serve:cert"
             },
-            "development": {
-              "devServerTarget": "angular-ssr-example:serve:development"
+            "dev": {
```

Comment:
I appreciate your clean naming style with use of "prod", "dev", "cert" and the like.